javascript/node/selenium-webdriver/firefox/index.js

```diff
@@ -367,14 +373,9 @@ class Driver extends webdriver.WebDriver {
       let service = createGeckoDriverService(binary);
       serverUrl = service.start();
       onQuit = () => service.kill();
+      caps.set(Capability.PROFILE, profile.encode());
```

Comment:
That is not currently possible.

However, @jgraham and I have been discussing the possibility of standardising certain common settings to share between browsers, such as binary location, flags, profile, preferences, and so on.

The argument to not ship profiles across the wire when they’re on the local file system is a strong contender, I think.

There is an [agenda item](https://www.w3.org/wiki/WebDriver/2016-July-F2F#Agenda) to discuss this at the next working group F2F in Seattle on 13-14 July.